include/rocksdb/utilities/backupable_db.h

```diff
@@ -91,7 +91,10 @@ struct BackupableDBOptions {
   // backups can come from different databases, hence a sst is not uniquely
   // identifed by its name, but by the triple (file name, crc32c, file length)
   // Default: false
-  // Note: this is an experimental option, and you'll need to set it manually
+  // Note: If this option is set to true, we recommend setting
+  // new_naming_for_backup_files to true as well. Otherwise, there is a
+  // non-negligible chance of filename collision when sharing tables in
+  // shared_checksum among several DBs.
```

Comment:
See below. The point of this enhancement is to make this option robust and no longer experimental.